app/driver.go

```diff
@@ -244,9 +257,13 @@ func deployFlexvolumeDriver(kubeClient *clientset.Clientset, c *cli.Context, man
 	}()
 
 	done := make(chan struct{})
+
```

Comment:
Need to `RegisterShutdownChannel` before starting the provisioner, in order for the signal to trigger `done`.

Reply:
Oh, it’s really a stupid mistake, thanks.